src/hotspot/share/runtime/globals.hpp

```diff
@@ -835,6 +835,11 @@ const int ObjectAlignmentInBytes = 8;
           "JVM aborts, producing an error log and core/mini dump, on the "  \
           "first occurrence of an out-of-memory error thrown from JVM")     \
                                                                             \
+  product(intx, WaitUserThreadAtExitTimeout, 300,                           \
+          "Maximum delay in milliseconds at exit waiting for user threads " \
+          "in native")                                                      \
+          range(0, max_intx)                                                \
```

Comment:
Given we are going to calculate the actual delay in 10ms chunks, then maybe this should be expressed as the number of 10ms chunks to wait? Otherwise 0-9 -> 0; 10-19->10; 20-29->20 etc.

And maybe we should limit the max to something big but sensible - like 10 seconds?

Reply:
Sounds good for limiting the max to 10 seconds. Changing to the value to the the number of 10ms chucks also makes sense to me. Thanks for thinking through that detail.

Reply:
With @dholmes-ora suggestion above, I think it probably makes sense to also change the flag to `UserThreadWaitAttemptsAtExit`.